src/public/modules/forms/admin/controllers/view-responses.client.controller.js

```diff
@@ -178,6 +184,31 @@ function ViewResponsesController(
     })
   }
 
+  vm.downloadAllAttachments = function() {
+    var zip = new JSZip();
+    let downloadPromises = []
+
+    for (const [questionNum, metadata] of vm.attachmentDownloadUrls) {
+      downloadPromises.push(fetch(metadata.url).then((response) => response.json()).then((data) => {
```

Comment:
could you use the AngularJS $http service instead of _fetch_ to make this GET request? it's not available on IE11.

Reply:
Changed to use `$http`.